### PR TITLE
Add ruby 2.7 preview

### LINKED ISSUE #4588: Add Ruby 2.7-preview to CI
We need to start testing against Ruby 2.7 as it will be released in December and contains a huge change that will likely affect InSpec.

https://bugs.ruby-lang.org/issues/14183

### PR DESCRIPTION
**Pull request by miah**
closes: #4588

### CODE REVIEW

**Review by clintoncwolfe — COMMENTED** [review]
The BK change looks fine, and certainly looks like it is doing the right thing according to its output. How do you want to handle the test fixes - on this PR?

**Review by zenspider — APPROVED** [review]
THANK YOU

**Comment by zenspider** [comment]
Hrm... Can we make this a soft failure for now? We obviously need to get these fixed asap. A lot of them are probably easy fixes but I also suspect there's going to be some dependency hell to fix.

**Review by zenspider — APPROVED** [review]
Hrm... Can we make this a soft failure for now?

We obviously need to get these fixed asap (but not on this PR). A lot of them are probably easy fixes but I also suspect there's going to be some dependency hell to fix.

**Comment by miah** [comment]
I'll tweak this so 2.7 is executed similar to our coverage test and allow failures.

File: .expeditor/verify.pipeline.yml [hunk 1]
```diff
@@ -33,6 +33,14 @@ steps:
       docker:
         image: ruby:2.6-stretch
 
+- label:  run-tests-ruby-2.7-preview
+  command:
+    - /workdir/.expeditor/buildkite/verify.sh
```
Comment (zenspider) [inline .expeditor/verify.pipeline.yml]: Probably should be `/workdir/.expeditor/buildkite/verify.sh || true` for now.

**Review by zenspider — CHANGES_REQUESTED** [review]

File: .expeditor/verify.pipeline.yml (hunk 1 above)
Comment (miah) [reply .expeditor/verify.pipeline.yml]: Fixed =)

**Comment by miah** [comment]
The 2.7 test will always pass for now.

We will remove the `|| true` when we get down to zero failures.

File issues and fixes to address the issues in the 2.7 bk step.

**Review by zenspider — APPROVED** [review]
Yay!

**Pull request merged**